tgui/packages/tgui/styles/interfaces/ClothingBooth.scss

```diff
@@ -1,31 +1,52 @@
-@use '.././components/Button.scss';
+@use '../base.scss';
+@use '../colors.scss';
 
-.clothingbooth__ {
-  &dropdown {
+.clothingbooth {
+  &__dropdown {
     margin-bottom: 0 !important;
     margin-right: 0 !important;
   }
 
-  &boothitem {
+  &__boothitem {
     padding-left: 0.5rem;
     padding-right: 0.5rem;
-  }
 
-  &boothitem > div {
-    pointer-events: none;
+    &:hover,
+    &--selected {
+      background-color: colors.$good;
+    }
   }
 
-  &boothitem:hover,
-  &boothitem-selected {
-    background-color: Button.$color-selected;
+  &__boothitem > div {
+    pointer-events: none;
   }
 
-  &boothitem:active {
-    background-color: lighten(Button.$color-selected, 30%);
+  &__boothitem:active {
+    background-color: lighten(colors.$good, 30%);
   }
 
-  &preview {
+  &__preview {
     width: 64px;
     height: 128px;
   }
+
+  &__swatch {
+    &_box {
+      margin: 0.2em;
+      &:hover {
+        outline: 0.167rem solid colors.$good;
+      }
+    }
+  }
+
+  &__groupingtag {
+      border-radius: base.$border-radius;
+  }
+}
+
+// If any swatch foreground shapes need custom styling, this is where it lives.
+.clothingbooth__swatch_icon {
+  &_rainbow {
+    background-image: linear-gradient(45deg, red,orange,yellow,green,blue,indigo,violet);
+  }
 }
```

Comment:
No real issue with this being its own little thing, but should still live within the top-level `.clothingbooth` "block". i.e. `&__swatch_icon` (and move into the block above).

Also, naming convention would be `swatch-icon`, not `swatch_icon`.